### PR TITLE
feat(source-intercom): add tickets stream 

### PR DESCRIPTION
**Pull request by sherifnada**
> [!note]
> Edited by @natikgadzhi 

## What
Adds a new `tickets` stream to Instagram source! <3 

## How
1. New stream
2. YOLO CDK to 2.0

## User Impact
- Adding new stream, no breaking changes for existing streams

## Can this PR be safely reverted and rolled back?
- [x] YES QUESTIONMARK
- [ ] NO ❌

## TODO

- [ ] Version bump 
- [ ] Test & Pray

### CODE REVIEW

File: airbyte-integrations/connectors/source-intercom/source_intercom/components.py [hunk 1]
```diff
@@ -26,7 +25,7 @@
 
 
 @dataclass
-class IncrementalSingleSliceCursor(Cursor):
+class IncrementalSingleSliceCursor(ResumableFullRefreshCursor):
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-intercom/source_intercom/components.py:28]: im pretty sure this is wrong

File: airbyte-integrations/connectors/source-intercom/source_intercom/manifest.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-version: 0.72.1
+version: 2.0.0
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-intercom/source_intercom/manifest.yaml:1]: Bold :D

**Review by natikgadzhi — COMMENTED** [review]
Thank you <3 

@girarda do you want us to take over Intercom, or is there any work in progress and you'd like to keep it in Python sources for now? I'm slightly lost and don't remember, but I think it's marketplace team now?

File: airbyte-integrations/connectors/source-intercom/source_intercom/manifest.yaml (hunk 2 above)
Comment (sherifnada) [reply airbyte-integrations/connectors/source-intercom/source_intercom/manifest.yaml:1]: bleeding edge or bust

**Pull request closed without merging**